.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  cpp:
+    image: humancompatibleai/goattack:cpp
+    build:
+      context: .. # Repo root
+      dockerfile: ./compose/cpp/Dockerfile
+      target: build-deps
+    volumes:
+      - type: bind
+        source: /nas/ucb/josephmiller/test-evaluate-loop/test3
```

Comment:
This seems specific to a particular use case, can we make it more generic?

Reply:
No I don't think so, for the same reason as above.